package.json

```diff
@@ -108,9 +108,40 @@
     "version-check": "node ./scripts/tasks/version-check.js"
   },
   "jest": {
-    "projects": [
```

Comment:
I tried keeping just one project but even the newest version of Jest (21 alpha) crashes with just one project so I had to inline the config back. cc @cpojer in case this is not known.